src/web/client/utilities/commonUtil.ts

```diff
@@ -56,6 +64,22 @@ export function GetFileContent(result: any, attributePath: IAttributePath) {
     return fileContent;
 }
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export function setFileContent(result: any, attributePath: IAttributePath, content: any[]){
+    if(attributePath.relativePath.length > 0){
+        const jsonFromOriginalContent = JSON.parse(
+            result[attributePath.source]
+        );
+
+        jsonFromOriginalContent[attributePath.relativePath] =
+            content;       
+        result[attributePath.source] = JSON.stringify(jsonFromOriginalContent);
+    }
```

Comment:
We should always keep JSON.stringify in try-catch block.

https://www.appmarq.com/public/security,1020872,Always-use-JSONparse-JSONstringify-with-trycatch-block-TypeScript